src/Symfony/Component/Messenger/Command/DebugMessengerCommand.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * A console command for messenger information.
+ *
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ *
+ * @experimental in 4.1
+ */
+class DebugMessengerCommand extends Command
+{
+    protected static $defaultName = 'debug:messenger';
+
+    private $mapping;
+
+    public function __construct(array $mapping)
+    {
+        parent::__construct();
+
+        $this->mapping = $mapping;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription('Lists classes you can dispatch using the message bus')
+            ->setHelp(<<<'EOF'
+The <info>%command.name%</info> command displays all classes that can be
+dispatched using the message bus:
+
+  <info>php %command.full_name%</info>
+
+EOF
+            )
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        $io->title('Message Bus');
+        $io->text('The following classes can be dispatch:');
+        $io->newLine();
+        $tableRows = array();
+        foreach ($this->mapping as $message => $handlers) {
+            $tableRows[] = array(sprintf('<fg=cyan>%s</fg=cyan>', $message));
+            foreach ($handlers as $handler) {
```

Comment:
I think that having the list of handlers in another column would make more sense and be more coherent with the rest of the `debug:` commands as well.

Reply:
well.. i took `debug:autowring` for inspiration. IMHO that one reads well :)

Also with multiple columns we probably run into a spacing issue, assuming all service IDs are FQCN.